File: week11_test/devowel_file.pl
#!/usr/bin/perl -w

use strict;

foreach my $arg (@ARGV){
	open F, '<', $arg or die "Cannot open $arg";
	my @a = <F>;
	close F;
	foreach my $line (@a){
		$line =~ s/[aeiou]//gi;
	}
	open my $b, '>', $arg or die "Cannot write to $arg";
	print $b @a;
	close $b;
}

File: week11_test/devowel_file.py
#!/usr/bin/python3

import sys, re

for file in sys.argv[1:]:
	f = open(file,'r')
	data = f.read()
	data = re.sub('[aeiouAEIOU]','',data)
	F = open(file,'w')
	F.write(data)
	f.close()
	F.close()

